core/domain/exp_domain.py

```diff
@@ -577,8 +731,11 @@ def validate(self, interaction, exp_param_specs_dict):
         self.outcome.validate()
 
     def get_classifier_rule_index(self):
-        """Returns the index of the classifier in the answer groups, or None
-        if it doesn't exist.
+        """Get the index of the classifier in the answer groups.
```

Comment:
Gets (for consistency with the other docstrings). 

Please also check that all other docstrings start with a third-person singular present tense verb -- I see some below that don't.